frontend/doc-site/04_Analyze-Public-Data/4_0-Analyze-Public-Data.mdx

```diff
@@ -6,11 +6,11 @@ In order to understand which tool will best meet your needs, it’s helpful to u
 
 ## THE HOSTED DATA EXPLORER
 
-The hosted data explorer allows you to *explore individual data sets* hosted on the CellxGene platform in UMAP form.
+The hosted data explorer allows you to _explore individual data sets_ hosted on the CellxGene platform in UMAP form.
```

Comment:
prettier, our opinonated formatter, prefers `_` for italics, and `*` for bolding